projects/kalidea/kaligraphi/src/lib/02-form/kal-select/kal-select.component.ts

```diff
@@ -120,22 +119,29 @@ export class KalSelectComponent
   private isPanelOpen: boolean;
 
   /**
-   * Store Subscriptions
+   * list of custom class added to select overlay
    */
-  private subscriptionsList: Subscription[] = [];
+  private _overlayClassList: string[] = [];
 
   /**
-   * list of custom class added to select overlay
+   * Store Subscription
    */
-  private _overlayClassList: string[] = [];
+  private optionsChangeSubscription: Subscription;
```

Comment:
optionsChangeSubscription = Subscription.Empty;

Reply:
we do not need this syntax anymore with operator `?` : ` this.optionsChangeSubscription?.unsubscribe();`